app/uk/gov/hmrc/eoricommoncomponent/frontend/domain/subscription/BusinessShortName.scala

```diff
@@ -18,8 +18,10 @@ package uk.gov.hmrc.eoricommoncomponent.frontend.domain.subscription
 
 import play.api.libs.json.Json
 
-case class NameIdDetails(name: String, id: String)
+case class BusinessShortName(shortNameProvided: Boolean, shortName: Option[String])
```

Comment:
This needs to be only in reg, so can be deleted

Reply:
This class is a part of SubscriptionDetails and I didn't want to remove this field form that class. I'll try to remove it and clean SubscriptionDetails class in separate commit